game/views.py

```
@@ -642,3 +664,35 @@ def stats_view(request):
+    
+    if not cron_secret or not secrets_module.compare_digest(expected, provided):
+        return JsonResponse({'error': 'Unauthorized'}, status=401)
+        
+    try:
+        deleted, resigned = cleanup_stale_games()
+        return JsonResponse({
+            'status': 'success',
+            'deleted_games': deleted,
+            'resigned_games': resigned
+        })
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)